assets/src/application/ui/forms/formModal/FormModalForm.tsx

```diff
@@ -23,6 +18,20 @@ const dataStyle: CSSProperties = {
 	backgroundColor: '#26203d',
 };
 
+const modalStyle: CSSProperties = {
+	boxSizing: 'border-box',
+	maxHeight: '90%',
+	maxWidth: '1200px',
+	minHeight: '50%',
+	minWidth: '320px',
+	width: '80%',
+	overflowY: 'scroll',
+	position: 'absolute' as 'absolute',
+	left: '50%',
+	top: '50%',
+	transform: 'translate(-50%, -50%)',
+};
+
```

Comment:
We need to discuss about this approach at some point.
Not related to this PR, though.

Reply:
Agreed